apps/airdrop/src/App/Claim/ClaimConfirmation/ClaimConfirmation.tsx

```diff
@@ -118,10 +118,13 @@ export const ClaimConfirmation: React.FC = () => {
   const isTrustedSetup = claimState?.type === "ts";
   const isClaimSectionActive = !isTrustedSetup || !!tsSignature;
   const airdropAddressValid = bech32mValidation("tnam", airdropAddress);
+  const airdropPubKeyValid = bech32mValidation("tpknam", airdropPubKey);
 
-  const isValidForm = Boolean(
-    airdropAddressValid && airdropPubKey && isToggleChecked
+  const isTopFormValid = Boolean(isTrustedSetup ? tsSignature : true);
```

Comment:
`isTopFormValid` looks like it's the same thing as `isClaimSectionActive` so you might be able to remove one of them.

Reply:
Thanks! Fixed :)